project/LibDependencies.scala

```diff
@@ -11,9 +11,10 @@ object LibDependencies {
 
   val compile: Seq[ModuleID] = dependencies(
     shared = Seq(
-      "com.typesafe.play"  %% "play"            % PlayVersion.current,
-      "com.typesafe.play"  %% "filters-helpers" % PlayVersion.current,
-      "uk.gov.hmrc.webjars" % "hmrc-frontend"   % hmrcFrontendVersion
+      "org.scala-lang.modules" %% "scala-collection-compat" % "2.10.0",
```

Comment:
this is needed for cross-compilation to 2.12 - it provides backports of the new collections